test/two-fer/two_fer_test.py

```diff
@@ -15,16 +14,31 @@
 if str(LIBRARY) not in sys.path:
     sys.path.insert(0, str(LIBRARY))
 
-from common import Exercise, Status, BaseExerciseTest
+from common import Analysis, BaseExerciseTest, BaseFeedback, Exercise, Summary
+
 
 USES_STRING_FORMAT = """
-def two_fer(name="you"):
+'''
+My Answer to Exercism Python Track Two-Fer Exercise.
+'''
+def two_fer(name='you'):
+    '''This function takes a name and prints out one for [name], one for me.
+
+    In the absence of a name, it prints you.'''
     return "One for {}, one for me.".format(name)
+    
```

Comment:
What's the advantage of adding these comments to all the test cases? I find it makes it a little harder to see the difference of each one.

Reply:
(Don't want to speak for Bethany, but I followed the discussion a bit and I guess these docstrings are idiomatic, so adding them ensures the tests actually test the analyzer to ignore/find/parse them)

Reply:
Yeah - I can totally see one test that checks for docstrings. But presuming this gets some wider test coverage for a couple of dozen solutions it seems that adding them to all of them is going to add a lot of "noise" when trying to visually parse the tests?

Reply:
> docstrings are idiomatic

Indeed they are

> adding them ensures the tests actually test the analyzer to ignore/find/parse them

This. Also, we have (or are going to have) a `docstrings` concept exercise, so the analyzer will need to be able to find/parse them

> it seems that adding them to all of them is going to add a lot of "noise" when trying to visually parse the tests?

I personally disagree; function and class docstrings are common enough in Python APIs that, to most Python developers, docstrings don't register as noise any more than `def` or `class` keywords; they are just a normal part of the syntax.

Reply:
OK. Seems strange to me. Having 7x more lines per example, with like 50 example, with no syntax highlighting to help you see through them, feels like a lot of extra noise for a brain to work through, especially in an analyzer where the differences between the test cases can be extremely subtle. But it's not my codebase so not something my opinion carries weight on. I'll mark this as resolved 🙂

Reply:
Chiming in late to the convo -- while DJ and Corey make some very good points about docstrings (_adding them can give you nice `help` messages and lightweight tests in the REPL for "free"_) --  and we do indeed intend to write one or more exercises about them -- 
the sad truth is that I added them into the tests to get `pylint` to **shut up already**.  😉    It kept failing even my CELEBRATORY code, which was causing bogus test case failure.  

I do find them very hard to read, especially with them defined as mulit-line constant strings in the test file.  So two things on that front:  
1. We may want to disable `pylint` warnings about docstrings on a per-exercise basis.  
2.  I am going to figure out some other method of making those test cases because that test file is making me really sad.  Current thought is that I would have a bunch of sample solutions in the test directory - which would also then test the reading/loading logic.  Something for the second refactor.  😄

Reply:
@BethanyG Cool 🙂 

- I'm quite pleased with the [Ruby approach](https://github.com/exercism/ruby-analyzer/blob/main/test/exercises/two_fer_test.rb) if you want some inspiration there: 
- @ErikSchierboom uses [fixtures instead in C#](https://github.com/exercism/csharp-analyzer/tree/main/test/Exercism.Analyzers.CSharp.IntegrationTests/Solutions), which is also nice

I also have disabled lots of rubocop checks for lots of this stuff as they're pretty unusual applications for it to work with.

Reply:
I have both inline solutions and fixture-files / directories where the latter have highlighting. Might be a cheap (*future PR) solution for the noise @BethanyG .